apache-spark-master/run-master.sh

```diff
@@ -0,0 +1,3 @@
+/opt/apache-spark/sbin/start-master.sh
+
+tail -f /opt/apache-spark/logs/* 1>&2
```

Comment:
So the question is if we want to run the container in the foreground or detached in the background, i.e. with -d.